### PR TITLE
TofuCraftReload翻译提交

### PR DESCRIPTION
**Pull request by mlbhc**

### CODE REVIEW

**Comment by luoxueyasha** [comment]
豆腐官汉有 射命丸政（@0999312 ）接手，建议先和那边沟通，再考虑要不要投过来

**Comment by mlbhc** [comment]
上周就发了邮件但没有收到回复,我也没有办法

**Comment by luoxueyasha** [comment]
建议直接b站或者qq找过去（？

**Comment by baguchan** [comment]
I heard you guys are working on a Chinese pullrequest for tofucraftreload
(sorry for english)

**Comment by mlbhc** [comment]
> I heard you guys are working on a Chinese pullrequest for tofucraftreload (sorry for english) As the current author of Tofu Craft Reload, is there anything I can do to help?
They said TofuCraftReload have an official Chinese translator
If I want pullrequest should I ask his permission?

**Comment by mlbhc** [comment]
> I heard you guys are working on a Chinese pullrequest for tofucraftreload (sorry for english) As the current stuff of Tofu Craft Reload, is there anything I can do to help?

They said TofuCraftReload have an official Chinese translator
If I want pullrequest should I ask his permission?

**Comment by baguchan** [comment]
> They said TofuCraftReload have an official Chinese translator If I want pullrequest should I ask his permission?

no need permission! go ahead!
Also, Mr.射命丸政 is currently entrusting the development to me.

**Comment by luoxueyasha** [comment]
So as @baguchan replied, this pull request will sooooooon be reviewed.
既然豆腐（原&现）作者baguchan回复了，这边也会尽快开始对应的review工作。

虽然我们更推荐直接投官库，无论是对接也好审核周期也好，那边都会更方便一些。

File: projects/1.20/assets/tofucraftreload/tofucraft/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+  "advancements.tofucraft.very_strange_soybeans.title": "非常奇怪的大豆",
+  "advancements.tofucraft.very_strange_soybeans.description": "找到灵魂大豆",
+  "advancements.tofucraft.tofu_mental.title": "有情感的豆腐",
+  "advancements.tofucraft.tofu_mental.description": "使用哭泣黑曜石压制豆腐",
+  "advancements.tofucraft.fermentation_basics.title": "发酵基础",
+  "advancements.tofucraft.fermentation_basics.description": "制作曲种，等待发酵",
+  "advancements.tofucraft.miso.title": "一个大豆;两种制品",
+  "advancements.tofucraft.miso.description": "和豆腐一样，通过在木桶上下放置石头，来制作味噌和酱油",
+  "advancements.tofucraft.sniffed_revenge.title": "嗅觉的复仇",
+  "advancements.tofucraft.sniffed_revenge.description": "为了报复嗅觉灵敏的监守者,把辣椒扔向它吧！",
```
Comment (luoxueyasha) [inline projects/1.20/assets/tofucraftreload/tofucraft/lang/zh_cn.json:529]:
```suggestion
  "advancements.tofucraft.sniffed_revenge.description": "为了报复嗅觉灵敏的监守者，把辣椒扔向它吧！",
```
我们遵循《标点符号规范》，需要全文使用中文全角符号。
https://cfpa.site/TransRules/
此类问题请全文通查

**Review by luoxueyasha — CHANGES_REQUESTED** [review]

File: projects/1.20/assets/tofucraftreload/tofucraft/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+  "tofucraft.tofu_book.tooltip" : "豆腐知识大全 2022",
+  "tofucraft.tofu_book.tooltip2" : "如果你想阅读这本书,请安装Patchouli(帕秋莉手册)mod!",
+  "item.tofucraft.tofu_book" : "豆腐匠人指南",
+  "tofucraft.jei.bittern" : "卤制",
+  "tofucraft.jei.harden" : "压制",
+  "tofucraft.need_defeat_boss" : "首先需要打败Boss",
+  "block.tofucraft.rice_block" : "大米包装",
+  "tofucraft.config.travelerTofunianSpawn" : "豆腐游商刷新",
+  "itemGroup.tofucraft.tofus.main_tab" : "豆腐",
+  "itemGroup.tofucraft.tofu_foods.main_tab" : "豆腐工艺:食物",
```
Comment (luoxueyasha) [inline projects/1.20/assets/tofucraftreload/tofucraft/lang/zh_cn.json:613]: 请仔细阅读《[翻译规范与指南](https://cfpa.site/TransRules/)》，这些相关规范与资料库在我们的readme和CONTRIBUTING里都有介绍

对于创造模式物品栏分栏，我们一般使用`丨`（汉字，gun3）进行分割
如果没有特殊情况，此类问题请全文通查

```suggestion
  "itemGroup.tofucraft.tofu_foods.main_tab" : "豆腐工艺丨食物",
```

**Review by luoxueyasha — CHANGES_REQUESTED** [review]

**Comment by luoxueyasha** [comment]
我最近加班，没时间，并且不要总是request review
可以去qq群里喊一声，找一下其他有空的审核

**Comment by mlbhc** [comment]
投官库了

**Pull request closed without merging**